test/functionality/dataflow/environments/resolve-tests.ts

```diff
@@ -42,4 +44,28 @@ describe('Resolve', () => {
 			});
 		});
 	});
+	describe('Builtin Constants', () => {
+		it('Unknown Identifier', () => {
+			const env = defaultEnv();
+			const result = resolvesToBuiltInConstant(undefined, env, undefined);
+			expect(result, 'should be Ternary.Never').to.be.equal(Ternary.Never);
+		});
+		it('Undefined Identifier', () => {
+			const env = defaultEnv();
+			const result = resolvesToBuiltInConstant('foo', env, undefined);
+			expect(result, 'should be Ternary.Never').to.be.equal(Ternary.Never);
+		});
+
+		const testSingle = (identifier: Identifier, expected: unknown) => function() {
```

Comment:
Usually we use such a style to include the `it` right away, just so we can theoretically abstract this into many `it`s later :D So could you refactor this into something like:

```ts
function test(label: string, identifier: Identifier, expected: unknown) {
  // ...
}
```